webui/react/src/pages/ExperimentList.tsx

```diff
@@ -398,12 +405,7 @@ const ExperimentList: React.FC = () => {
         render: checkmarkRenderer,
         title: 'Archived',
       },
-      {
-        dataIndex: 'forkedFrom',
-        key: 'forkedFrom',
-        render: forkedFromRenderer,
-        title: 'Forked From',
-      },
+
```

Comment:
just a nitpick, but you can remove this space

Reply:
Ok I made change